### DISCUSSION TITLE
What to expect from Colang 2.0

### DISCUSSION THREAD

**Discussion by habanoz** [NVIDIA-NeMo/Guardrails#421] [Q&A] [answered]
Hello,

First and foremost, I want to present my sincere thanks to the team (to the contributors as well) for this beautiful work.

That being said, I am curious about what to expect from Colang 2.0. When will it be available? At this point, documentation does not include anything about colang 2.0.

Thanks
Regards

**Comment by drazvan** [comment, marked as answer]
Thanks for the message @habanoz! An initial documentation for Colang 2.0 will be added with the next release (`0.9.0`) which is scheduled for early next week. For the next 2-3 releases, it will be marked as experimental, and at some point, we'll deprecate Colang 1.0.

**Reply by habanoz** [reply]
Thanks @drazvan!
